frontend/src/contexts/agreement.tsx

```diff
@@ -15,13 +16,46 @@ const AgreementProvider: FC = () => {
       agreementId,
     },
   });
+
+  const [voteFn, { error: voteError }] = useMutation(voteMutation, {
+    refetchQueries: ['votes'],
```

Comment:
This is meaningless at this point, because we don't have a query called 'votes' (not yet at least)

Reply:
removed